event_badge_design/views/event_event.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_event_form" model="ir.ui.view">
+    <field name="model">event.event</field>
+    <field name="inherit_id" ref="event.view_event_form" />
+    <field name="arch" type="xml">
+        <notebook position="inside">
+            <page name="badge_front" string="Badge front">
```

Comment:
Missing field event_logo?

Reply:
ah right, added it on the badge front tab